app_dart/test/service/scheduler/graph_test.dart

```diff
@@ -179,4 +179,93 @@ targets:
           ));
     });
   });
+
+  group('scheduler config tested and compared with tip of tree builders', () {
+    late SchedulerConfig? totConfig;
+
+    setUp(() {
+      YamlMap? totYaml = loadYaml('''
+enabled_branches:
+  - master
+targets:
+  - name: A
+  - name: B
```

Comment:
Try to keep tests as simple as possible.

In this case, we only need 1 target to validate the functionality